client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -175,3 +190,16 @@ def get_selection(self):
             )
 
         return selection
+
+
+class createStaticMesh(CreateFBX):
+    """Static Meshes as FBX. """
+
+    identifier = "io.openpype.creators.houdini.staticmesh.fbx"
+    label = "staticMesh (FBX)A"
+    product_type = "staticMesh"
+    icon = "cube"
```

Comment:
What's the reason for the icon change?
```suggestion
    icon = "fa5s.cubes"
```

Reply:
@BigRoy the model abc also had cube as icon, so i think i copied it from there.